src/libraries/System.Net.Security/src/System/Net/Security/SslStreamPal.Windows.cs

```diff
@@ -15,6 +15,12 @@ namespace System.Net.Security
 {
     internal static class SslStreamPal
     {
+        private static readonly bool UseNewCryptoApi =
+            // On newer Windows version we use new API to get TLS1.3.
+            // API is supported since Windows 10 1809 (17763)but there is no reason to use at the moment.
+            Interop.NtDll.RtlGetVersionEx(out Interop.NtDll.RTL_OSVERSIONINFOEX osvi) == 0 &&
+            osvi.dwMajorVersion >= 10 && osvi.dwBuildNumber >= 18836;
```

Comment:
We fixed `Environment.OSVersion` in https://github.com/dotnet/runtime/pull/33651, so no need to call this directly now.